planning march 2021/userStoriesSummary.md

```diff
@@ -0,0 +1,145 @@
+- [ ] Reviewer with consolidation permissions can start a consolidation when review are available for consolidation - 1
+- [ ] Consolidator is able to agree or disagree with reviewer suggestions and make comments - 1
+- [ ] Consolidator is able submit consolidation as changes requested (when at least on disagree) - 1
+- [ ] Consolidator is able to submit consolidation as LOQ (at least one declined), Non Conform (at least on decline), Conform (all approved and agreed) - 1
+- [ ] See a list of applications that I am permitted to review - 1 (duplicate)
+- [ ] List shows applications that have been assigned to review - 1 (duplicate)
+- [ ] Review home page shows status and progress of associated reviews (and what I can do with each one) - 1 (duplicate?)
+- [ ] Can "self-assign" review if application and reviewLevel allows it - 1
+- [ ] Select which comments get sent to applicant in LOQ (if Level 2+) - 3
+- [ ] Get notified of new jobs -- assignments or changes submitted - 2 (duplicate)
+- [ ] Apply for permissions to perform new tasks/reviews - 3
+- [ ] Make changes to my reviews on request from a senior reviewer - 1
+- [ ] If senior reviewer, make Final decision for application (same as consolidator) - 1
+- [ ] Senior reviewer can make Final decision regardless of lower level recommendations (i.e. can overrule earlier reviews) - 3
+- [ ] When drug registration/import-permit/company is approved generates a certificate/permit/company - 1
+- [ ] When drug registration/import-permit/company is declined, inform user of non conformity with message and generated document - 1
+- [ ] When consolidation/review (highest level) is submitted as LOQ, inform user of LOQ and generate LOQ document - 1
+
+### Assigner
+
+- [ ] Restrict visibility of applications where assigner is able to assign (for current stage) - 1
+- [ ] See actionables in the list of applications to assign or re-assign - 1(assign) 2(re-assign)
+- [ ] See review home page with assignment of reviewers per section and progress if already submitted - 1
+- [ ] Per section, show which reviewers can be assigned (based on the configuration) - 1
+- [ ] Able to assign in review home page by section to user - 1
+- [ ] Get prompt before re-assigning to other reviewer in case review has been started or submitted - 2
+- [ ] See assignment done by others when higher level permissions apply - 3
+- [ ] Get notified by email when new assignment task is required for an application (new application submitted, change of stage) - 2
+- [ ] Able to configure assigner for a stage and level 1 - 1
+- [ ] Able to configure assigner for a stage and any level - 3
+- [ ] Restrict reviewer configurations for reviewing a set of sections - 2
+- [ ] Ability to set level of notifications - 3
+- [ ] Can assign by question - 3
+
+### Public user
+
+- [ ] View drug registrations (API)
+- [ ] Create an account on the system 1d
+- [ ] I can confirm my email address after creating user account
+
+### Admin
```

Comment:
I have quite a few more Admin tasks in mind:

- [ ] Update permissions associated with user/s
- [ ] Upload templates for document generation and configure at what 'level', 'stage', 'outcome', 'role' and 'template' this documents should apply
- [ ] Able to create new draft version of application template and edit it without affecting original 'live' version
- [ ] Add/remove/edit - sections, stage and review levels
- [ ] Add/remove/edit - Form elements (is visible, is required, is editable, parameters)
- [ ] Add/remove/edit - Actions
- [ ] Add/remove/edit - Template Permissions
- [ ] Edit expiry and auto declined periods of applications for a template
- [ ] Import/Export - Whole template
- [ ] Import/Export - Template structure
- [ ] Import/Export - Selected form elements
- [ ] Import/Export - Actions
- [ ] Import/Export - Permission Structure
- [ ] Able to edit permission policies
- [ ] Able to edit permission names
- [ ] Can setup outcome tables and configure how they are populated (actions)
- [ ] Can expose outcome tables (partial or as a whole) to public users